### PR TITLE
add flag to support custom number of virtualbox processors

### PR DESCRIPTION
**Pull request by greentg**
This is an open feature request at #819 
Presently there are several other providers that expose this as a tunable, so there's an argument to be made that it should be included for consistency. 

The default behavior is to continue using the number of the CPUs on the machine of the flag isn't specified.

Flag is: --virtualbox-processors

### CODE REVIEW

**Comment by stephenlawrence** [comment]
I was putting together a pull request for this, you beat me to it. :)

File: drivers/virtualbox/virtualbox.go [hunk 1]
```diff
@@ -58,11 +59,24 @@ func init() {
...
+	return cpus
+}
+
 // RegisterCreateFlags registers the flags this driver adds to
 // "docker hosts create"
 func GetCreateFlags() []cli.Flag {
 	return []cli.Flag{
 		cli.IntFlag{
+			Name:  "virtualbox-processors",
+			Usage: "Number of virtual CPUs cores for host",
```
Comment (hairyhenderson) [inline drivers/virtualbox/virtualbox.go]: Should probably mention that it defaults to the number of cores on the host. The default might seem to make it obvious, but it's easy to miss the correlation ;)

Comment (greentg) [inline drivers/virtualbox/virtualbox.go]: Thanks @hairyhenderson, added a description for default number of cores.

**Comment by ehazlett** [comment]
Sorry we had another submission and I forgot about this one!  Closing as the functionality has been merged.  Thanks for the contribution!

**Pull request closed without merging**